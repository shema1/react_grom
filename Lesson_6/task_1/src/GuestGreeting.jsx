import React, { Component } from "react";

const GuestGreeting = () => {
  return <h1>Hi Strannger</h1>;
};

export default  GuestGreeting
