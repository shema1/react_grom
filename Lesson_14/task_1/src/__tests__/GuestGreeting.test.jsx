import React from "react";
import { shallow, mount } from 'enzyme'
import GuestGreeting from "../GuestGreeting";

describe('UserGreeting', () => {
  it('should display user greeting', () => {
    const wrappedComponent = shallow(<GuestGreeting/>)
    expect(wrappedComponent).toMatchSnapshot();
  });
});