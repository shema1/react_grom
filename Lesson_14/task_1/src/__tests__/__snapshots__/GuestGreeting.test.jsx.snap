// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserGreeting should display user greeting 1`] = `
<h1>
  Hi stranger âœŒ Please login ðŸš€
</h1>
`;
